### ISSUE TITLE
Many included examples no longer run correctly

### ISSUE THREAD

**Issue by mhetrick** [supercollider/supercollider#3731]
A number of files in the "examples" directory (included in the distribution) no longer run correctly or could use improvements to be more beginner friendly. I am seeing this occur in the Mac 3.9.3 distribution. I will compare against the Windows distribution later today.

---
(Edited out a number of examples that weren't loading due to my inexperience with ProxySpace)
---

"demonstrations/GetTheTwits" requires the XML Quark, but doesn't have an "include" to quickly download it.
"demonstrations/HarmonicsVoice" is a tutorial in HTML format, which is different from every other scd file in that folder. It might make sense just to turn it into an scd file and convert the text to comments.
"demonstrations/sc_onliner" should be renamed to "sc_oneliner".

"demonstrations/Atari2600" is an interesting one, as it depends on the user manually downloading a plugin. It's a good example, but it might not make sense to include it in the standard distribution (especially because it appears second alphabetically in the "demonstrations" folder).

"GUI examples/ScopeExample.scd" has "GUI.cocoa", which no longer works. "GUI.swing" throws a deprecation warning. The rest of the example throws a "message 'ar' not understood" error.

"other/keepyuppy" runs, but throws an error: "Primitive '_QObject_SetEventHandlerEnabled' failed.". It doesn't seem to respond to keyboard input.

**Comment by patrickdupuis** [comment]
Thanks for this @mhetrick. Indeed, we need to go through the examples and fix the ones that are broken. I'll bring it up at our next dev meeting.

**Comment by telephon** [comment]
Works fine here. I think you have probably pushed a `ProxySpace`, this is why you get the above error message.

**Comment by mhetrick** [comment]
@telephon Yes doing a full scide restart fixed Termite for me. I'm not sure if the issue should be closed, though, due to the different (non ProxySpace) failures in ScopeExample and keepyuppy. I'm editing my above post to remove the ProxySpace errors.

**Comment by telephon** [comment]
good!
